test/install-preflight.test.ts

```
@@ -1334,7 +1343,7 @@ describe("installer release-tag resolution", () => {
    * Requires the source guard so that main() doesn't run on source.
    * `fakeBin` must contain a `curl` stub (and optionally `node`).
    */
-  function callResolveReleaseTag(fakeBin, env = {}) {
+  function callResolveReleaseTag(fakeBin: string, env: NodeJS.ProcessEnv = {}) {
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)